package.json

```diff
@@ -17,9 +17,15 @@
   },
   "dependencies": {
     "@t3-oss/env-nextjs": "^0.12.0",
+    "class-variance-authority": "^0.7.1",
+    "clsx": "^2.1.1",
+    "lucide-react": "^0.543.0",
     "next": "^15.2.3",
     "react": "^19.0.0",
     "react-dom": "^19.0.0",
+    "react-icons": "^5.5.0",
```

Comment:
Please use only one icon provider. Maybe we should use Lucide. So, please remove react-icons then find some alternative from Lucide.